src/main/java/cz/cvut/kbss/study/model/Role.java

```diff
@@ -0,0 +1,113 @@
+package cz.cvut.kbss.study.model;
+
+import cz.cvut.kbss.jopa.model.annotations.Individual;
+import java.util.Optional;
+import org.apache.poi.ss.formula.atp.Switch;
+
+public enum Role {
+
+    // TODO deprecated -- should be removed.
+    @Individual(iri=Vocabulary.s_i_administrator)
+    administrator(Vocabulary.s_i_administrator),
```

Comment:
Maybe consider to have something like:
administrator(SecurityRole.ADMIN, Vocabulary.s_i_administrator),